Problem1.java

```diff
@@ -0,0 +1,34 @@
+// Time Complexity : Need help
```

Comment:
O(n) where n is the number of elements is the time complexity for your code as you are iterating over all the elements of the tree!!